tests/run.sh

```diff
@@ -12,7 +12,7 @@ SMOKE_TEST_OUTPUT="Never executed. Problem with one of previous stages"
 [ -z ${LICENSE_PATH+x} ] && (echo "Please set LICENSE_PATH"; exit 1)
 [ -z ${PULL_SECRET_PATH+x} ] && (echo "Please set PULL_SECRET_PATH"; exit 1)
 [ -z ${DOMAIN+x} ] && DOMAIN="tectonic-ci.de"
-[ -z ${AWS_REGION+x} ] && AWS_REGION="eu-west-1"
+[ -z ${AWS_REGION+x} ] && AWS_REGION="us-west-1"
```

Comment:
There are a few other occurrences of `eu-west-1`:

```console
$ git grep eu-west origin/pr/89
origin/pr/89:examples/tectonic.aws.yaml:    # `{ eu-west-1a = "10.0.0.0/20", eu-west-1b = "10.0.16.0/20" }`
origin/pr/89:examples/tectonic.aws.yaml:  region: eu-west-1
origin/pr/89:examples/tectonic.aws.yaml:    # Example: `{ eu-west-1a = "10.0.64.0/20", eu-west-1b = "10.0.80.0/20" }`
origin/pr/89:installer/pkg/config-generator/fixtures/test-aws.yaml:  region: eu-west-1
origin/pr/89:installer/pkg/config/aws/aws.go:   DefaultRegion = "eu-west-1"
origin/pr/89:installer/pkg/workflow/fixtures/terraform.tfvars:  "tectonic_aws_region": "eu-west-1",
origin/pr/89:steps/variables-aws.tf:`{ eu-west-1a = "10.0.0.0/20", eu-west-1b = "10.0.16.0/20" }`
origin/pr/89:steps/variables-aws.tf:Example: `{ eu-west-1a = "10.0.64.0/20", eu-west-1b = "10.0.80.0/20" }`
origin/pr/89:tests/jenkins-jobs/maintenance/tag_clean_aws_grafiti_job.groovy:    "ap-northeast-1" "eu-central-1" "eu-west-1" "eu-west-2" "sa-east-1" )
```

You may want to update at least the the `installer/` instances as well to stay consistent.